test/controllers/declaration/CommodityMeasureControllerSpec.scala

```diff
@@ -35,16 +35,16 @@ import uk.gov.hmrc.auth.core.InsufficientEnrolments
 class CommodityMeasureControllerSpec
     extends CustomExportsBaseSpec with CommodityMeasureMessages with Generators with PropertyChecks with OptionValues {
 
-  val uri = uriWithContextPath("/declaration/commodity-measure")
-  val form = CommodityMeasure.form()
+  private val uri = uriWithContextPath("/declaration/commodity-measure")
```

Comment:
:+1: 

We keep forgetting doing that. In `Java` I was so more careful. Good spot